### PR TITLE
Ests 136905 - Protection domain creation is not required if no valid protection is found

### PR DESCRIPTION
**Pull request by tghosh0202**

### CODE REVIEW

File: dne-paqx/src/main/java/com/dell/cpsd/paqx/dne/repository/H2DataRepository.java [hunk 1]
```diff
@@ -818,6 +818,31 @@ public DiscoveredNodeInfo getDiscoveredNodeInfo(String uuid)
 
     @Override
     @Transactional
+    public ScaleIOProtectionDomain createProtectionDomain(String jobId, String protectionDomainId, String protectionDomainName)
```
Comment (kushagra-mahajan) [inline dne-paqx/src/main/java/com/dell/cpsd/paqx/dne/repository/H2DataRepository.java]: Do we need this now?

File: dne-paqx/src/main/java/com/dell/cpsd/paqx/dne/service/task/handler/preprocess/FindProtectionDomainTaskHandler.java [hunk 2]
```diff
@@ -135,28 +137,11 @@ public boolean executeTask(Job job)
...
-
-                if (StringUtils.isEmpty(protectionDomainId))
-                {
-                    throw new IllegalStateException("Unable to Create Protection Domain with name: " + protectionDomainName);
-                }
-
-                response.setProtectionDomainName(protectionDomainName);
-                response.setProtectionDomainId(protectionDomainId);
+                //createProtectionDomain(job, response, scaleIo); //Disabled creation of new protection domain ESTS-136904
+                LOGGER.info("No protection domain found. Creation of protection domain is not required.");
```
Comment (kushagra-mahajan) [inline dne-paqx/src/main/java/com/dell/cpsd/paqx/dne/service/task/handler/preprocess/FindProtectionDomainTaskHandler.java]: log.error?

File: dne-paqx/src/main/java/com/dell/cpsd/paqx/dne/repository/H2DataRepository.java (hunk 1 above)
Comment (tghosh0202) [reply dne-paqx/src/main/java/com/dell/cpsd/paqx/dne/repository/H2DataRepository.java]: This is not needed now , may be required during multinode.

File: dne-paqx/src/main/java/com/dell/cpsd/paqx/dne/service/task/handler/preprocess/FindProtectionDomainTaskHandler.java (hunk 2 above)
Comment (tghosh0202) [reply dne-paqx/src/main/java/com/dell/cpsd/paqx/dne/service/task/handler/preprocess/FindProtectionDomainTaskHandler.java]: Okay I'll change

**Pull request merged**